tests/integration/PackExtractTest.php

```diff
@@ -30,7 +30,7 @@ public function archiveTypeProvider()
         if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
             return [['zip']];
         }
-        return [['zip'], ['tar'], ['tar.gz'], ['tar.bz2'], ['tgz']];
+        return [['tar'], ['tar'], ['tar.gz'], ['tar.bz2'], ['tgz']];
```

Comment:
Was this change in error?

Reply:
oh crap :(

Reply:
Actually it's not just a typo: it raises an issue with the zip and "tar" exclude pattern formats. I had to made some more tests and maybe I'll have to add checks or conversions...

Reply:
I fix the typo. I had to normalize the exclude pattern and narrow it to regex without enclosing slashes so it's compatible with both Symfony Finder and Archive_tar